docs/storage/lumio/connect-local.md

```diff
@@ -0,0 +1,34 @@
+# Using LUMI-O without connecting to LUMI
```

Comment:
I would combine this page with "simple upload and download". It is basically the same, requires extra steps for configuration lumi-o modules can do on LUMI.

Reply:
Since no further comments, marking this as resolved